Source/Scene/Cesium3DTileset.js

```diff
@@ -1838,10 +1862,10 @@ define([
 
     var scratchCartesian2 = new Cartesian3();
 
-    function updateGeometricErrorLabels(tileset, frameState) {
+    function updateTileInfoLabels(tileset, frameState) {
```

Comment:
`TileInfo` is very generic; we try to avoid names like "info."  Perhaps `updateTileDebugLabels`.

Same for the `_tileInfoLabels` naming.